.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -10,8 +10,7 @@ assignees: ''
 ## Use case
 
 <!--
-     Please tell us the problem you are running into that led to you wanting
-     a new feature.
+     Please tell us what that led to you wanting a new feature.
```

Comment:
Maybe change to "Please provide some background information on your use case for this new feature" or something like that